src/wattsi.pas

```diff
@@ -2266,7 +2280,8 @@ function Main(): Boolean;
                // gen...
                for Variant in TVariants do
                begin
-                  Writeln('Generating ', Variant, ' variant...');
+                  if (not(Quiet)) then
+                     Writeln('Generating ', Variant, ' variant...');
```

Comment:
if you use Log() then this would be come `Log('Generating ' + VariantNames[Variant] + ' variant...');`

...where `VariantNames` is a constant defined as:

```delphi
const
   VariantNames: array[TVariant] = ('HTML', 'dev');
```
...or something like that.